uber/automated_emails.py

```diff
@@ -72,8 +72,8 @@ def __init__(self, subject, template, filter, **kwargs):
 
 
 class GuestEmail(AutomatedEmail):
```

Comment:
Since ``needs_approval=True`` is now already set to ``True`` there's no need to make it true in the subclasses.